tests/zfs-tests/cmd/largest_file/largest_file.c

```diff
@@ -64,7 +64,7 @@ main(int argc, char **argv)
 	offset_t	llseek_ret = 0;
 	int		write_ret = 0;
 	int		err = 0;
-	char		mybuf[5];
+	char		mybuf[5] = {0};
```

Comment:
Since this byte gets appended to a file I'd suggest initializing it with a pattern like "abcd\0".  This will make it easier to    spot check with a text editor if we ever need too.  It's also a little weird this array is bigger than it needs to be but let's just leave that as is.  It's no doing any harm.